apps/src/aichat/views/ChatMessageView.tsx

```diff
@@ -0,0 +1,80 @@
+import React, {useMemo, useState} from 'react';
+
+import ChatMessage from '@cdo/apps/aiComponentLibrary/chatMessage/ChatMessage';
+import {Role} from '@cdo/apps/aiComponentLibrary/chatMessage/types';
+import {getChatMessageDisplayText} from '@cdo/apps/aiComponentLibrary/chatMessage/utils';
+import Button from '@cdo/apps/componentLibrary/button/Button';
+import {AiInteractionStatus as Status} from '@cdo/generated-scripts/sharedConstants';
+
+import aichatI18n from '../locale';
+import {ChatMessage as ChatMessageType} from '../types';
+
+import TeacherFeedbackFooter from './TeacherFeedbackFooter';
+
+import moduleStyles from '@cdo/apps/aiComponentLibrary/chatMessage/chat-message.module.scss';
+
+interface ChatMessageViewProps {
+  chatMessage: ChatMessageType;
+  isChatHistoryView: boolean;
+}
+
+const ChatMessageView: React.FunctionComponent<ChatMessageViewProps> = ({
+  chatMessage,
+  isChatHistoryView,
+}) => {
+  const [showProfaneUserMessage, setShowProfaneUserMessage] = useState(false);
+
+  const getDisplayText: string = useMemo(() => {
```

Comment:
nit: maybe just `displayText` since it's a value? got confused for a sec and thought this was a function